app/src/etc/supervisord.conf

```diff
@@ -46,7 +46,17 @@ command = node NodeJSServer.js
 directory=/home/bdt/map/src/map_nodejs_server/
 stdout_logfile=/var/log/NodeJSServer.log
 stderr_logfile=/var/log/NodeJSServer.log
-autostart=false                ; start at supervisord start (default: true)
+autostart=true                 ; start at supervisord start (default: true)
 autorestart=false              ; restart at unexpected quit (default: true)
 stopasgroup=true               ; stop all subprocesses together
 killasgroup=true               ; kill all subprocesses together
+
+[program:startup]
+command = /bin/bash /home/bdt/map/src/map_scripts/startup_after_shutdown.sh
```

Comment:
I see, thats a cool approach. Thanks for suggesting that.

The point of the startup script I was trying to get supervisor to run is to reset a database table's auto increment field. After thinking over the day, it would be much easier to switch the database engine to MyISAM so that the problem doesn't exist in the first place - then theres no need for a startup script. The fewer scripts and things to think about, the better. Since we're going to go with that approach, I'm going to update this PR to remove the startup program. If we need a startup script for something else in the future we can revisit your suggestion.